### ISSUE TITLE
Traffic en temps réel toujours en "traffic normal"

### ISSUE THREAD

**Issue by lavoituredrole04** [pgrimaud/horaires-ratp-api#111]
J'ai remarqué que depuis quelques temps, toutes les lignes de trams/métro/RER ont le statut trafic "Trafic normal sur l'ensemble de la ligne" même lors de travaux, perturbations ou interruptions de trafic
https://api-ratp.pierre-grimaud.fr/v4/traffic

**Comment by FredericPuteaux** [comment]
Oui, les API de la RATP ne sont plus disponibles. La migration a été réalisée cet été sur IDF Mobilité. (https://data.iledefrance-mobilites.fr/api/v1/console/datasets/1.0/search/)
Le service ne peut donc plus fonctionner sans l'adapter.

**Comment by racso-dev** [comment]
@pgrimaud Est-ce que le projet est tjrs maintenu ? Est-ce que ça va être fix ?
